pkg/subctl/cmd/cloud/cleanup/aws.go

```diff
@@ -40,7 +40,7 @@ func newAWSCleanupCommand(restConfigProducer restconfig.Producer) *cobra.Command
 		Run: func(cmd *cobra.Command, args []string) {
 			status := cli.NewReporter()
 
-			err := cleanup.AWS(&restConfigProducer, status)
+			err := cleanup.AWS(&restConfigProducer, &config, status)
```

Comment:
The `config` struct fields `infraID, region, profile, credentialsFile, ocpMetadataFile` need to populated from the flag. And `config.GwInstanceType` to be passed as empty string.

Reply:
That's the case from what I see. What do you see different?

Reply:
I would have explicitly made `config.GwInstanceType = ""`

Reply:
No need - empty string is the default.